planning/behavior_path_planner/include/behavior_path_planner/scene_module/side_shift/side_shift_module.hpp

```diff
@@ -107,6 +132,8 @@ class SideShiftModule : public SceneModuleInterface
   inline PoseStamped getEgoPose() const { return *(planner_data_->self_pose); }
   PathWithLaneId calcCenterLinePath(
     const std::shared_ptr<const PlannerData> & planner_data, const PoseStamped & pose) const;
+
+  SideShiftRequestTimer request_time;
```

Comment:
```suggestion
  SideShiftRequestTimer request_timer_;
```
It is private var :+1: